symbiflow-vtr-optimized/build.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [ x"$TRAVIS" = xtrue ]; then
+	CPU_COUNT=2
+fi
+
+echo "============================================================"
+echo "CFLAGS='$CFLAGS'"
+echo "CXXFLAGS='$CXXFLAGS'"
+echo "CPPFLAGS='$CPPFLAGS'"
+echo "DEBUG_CXXFLAGS='$DEBUG_CXXFLAGS'"
+echo "DEBUG_CPPFLAGS='$DEBUG_CPPFLAGS'"
+echo "LDFLAGS='$LDFLAGS'"
+echo "------------------------------------------------------------"
+# -Wundef causes warnings on undefined preprocessor defines (e.g. o
+# tbb_config.h:56:7: warning: "__GLIBCPP__" is not defined, evaluates to 0 [-Wundef]
+# elif __GLIBCPP__ || __GLIBCXX__
+export CFLAGS="$(echo $CFLAGS | sed -e's/-Wundef //') -w"
+export CXXFLAGS="$(echo $CXXFLAGS | sed -e's/-Wundef //') -w"
+export CPPFLAGS="$(echo $CPPFLAGS | sed -e's/-Wundef //')"
+export DEBUG_CXXFLAGS="$(echo $DEBUG_CXXFLAGS | sed -e's/-Wundef //') -w"
+export DEBUG_CPPFLAGS="$(echo $DEBUG_CPPFLAGS | sed -e's/-Wundef //')"
+echo "CFLAGS='$CFLAGS'"
+echo "CXXFLAGS='$CXXFLAGS'"
+echo "CPPFLAGS='$CPPFLAGS'"
+echo "DEBUG_CXXFLAGS='$DEBUG_CXXFLAGS'"
+echo "DEBUG_CPPFLAGS='$DEBUG_CPPFLAGS'"
+echo "LDFLAGS='$LDFLAGS'"
+env
+
+# Where to get the SymbiFlow architecture install package.
+SYMBIFLOW_INSTALL_URL=https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/presubmit/install/763/20201002-081640/symbiflow-arch-defs-install-cd3b2f35.tar.xz
+BUILD_ROOT=$PWD
+
+# This will take a while to download, so fork it and rejoin later.
+mkdir symbiflow
+(
+    curl $SYMBIFLOW_INSTALL_URL | tar xJ -C symbiflow
+) &
+FETCH_SYMBIFLOW_PID=$!
+
+# Can't be added to source/patches in meta.yaml due to: https://github.com/conda/conda-build/issues/4118
+patch -p1 < vpr_fix_pgo.patch # Until merged: https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/1586
```

Comment:
This workaround could be used for other packages until https://github.com/conda/conda-build/issues/4118 is resolved.

Reply:
Why do we need to patch this at all?  It has been merged upstream?

Reply:
It hasn't been merged into SymbiFlow/vtr-verilog-to-routing yet, though. This is temporary to avoid blocking on that.

Reply:
Part of the work is getting these fixes merged on SymbiFlow/vtr-verilog-to-routing?  I haven't seen you open a PR on SymbiFlow/vtr-verilog-to-routing integrating the upstream changes, so I'm confused by what you are doing here.

Reply:
I just submitted a PR: https://github.com/SymbiFlow/vtr-verilog-to-routing/pull/558

But it won't work even after that is merged, because master+wip is broken.

Reply:
> because master+wip is broken.

Is it though?  I just linked you to a CI showing it wasn't.  If you believe differently, please fix the issue or fill an issue showing the problem.

Reply:
I think I need a new eblif now that VPR has been fixed.

Reply:
I've updated the eblif and removed this patch.